README.md

```diff
@@ -96,6 +96,29 @@ $ sudo cat /sys/kernel/debug/tracing/trace_pipe
            <...>-461101 [018] d... 505434.345367: bpf_trace_printk: UPROBE EXIT: return = 5
 ```
 
+## USDT
+
+`usdt` is an example of dealing with USDT probe which requires `systemtap-sdt-devel`
```

Comment:
I'm a bit worried that we require extra package here.  How about instead of defining our own USDT, example program would be attaching to existing USDTs in some well-known binary or shared library? Can you check if glibc, or bash, or something that's with high likelihood is going to be present on end user's system has some USDTs we can attach to?

Reply:
@chenhengqi  any updates on this?

Reply:
Updated. PTAL.